pulser/simulation/simulation.py

```diff
@@ -195,15 +336,14 @@ def _build_basis_and_op_matrices(self) -> None:
             self.basis_name = 'all'  # All three states
             self.dim = 3
             basis = ['r', 'g', 'h']
-            projectors = ['gr', 'hg', 'rr', 'gg', 'hh']
+            projectors = ['gr', 'hg', 'rr', 'gg', 'hh', 'hr']
```

Comment:
What's the `hr` projector for?

Reply:
No idea, I don't think we use this ? I'll remove it.